src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -57,8 +57,9 @@ void LottieLoader::release()
         free((char*)content);
         content = nullptr;
     }
-    free(dirName);
-    dirName = nullptr;
+    // FIXME: Commented out because image slot needs the dirName
```

Comment:
Memory leak. this is a regression, correct logic is required. probably, you can free in the destructor.

Reply:
ah yeah, i knew there. I have pushed so late.

For now I moved freeing to the destructor, and squashed meaningless commits